astropy/samp/tests/test_client.py

```diff
@@ -35,7 +35,8 @@ def samp_hub(request):
     """A fixture that can be used by client tests that require a HUB."""
     my_hub = SAMPHubServer()
     my_hub.start()
-    request.addfinalizer(my_hub.stop)
```

Comment:
This was the only line that used `request`, so `samp_hub` shouldn't be asking for it anymore.